scripts/hid_configurator/README.rst

```diff
@@ -82,7 +82,17 @@ Complete the following steps:
        In versions earlier than 5.44, the HID device attached by BlueZ could obtain wrong VID and PID values (ignoring values in Device Information Service), which would stop HIDAPI from opening the device.
        In versions earlier than 5.56, the HID device attached by BlueZ might provide incomplete HID feature report on get operation.
 
-   Additionally, to call the Python script on Linux without root rights, install the provided udev rule :file:`99-hid.rules` file by copying it to :file:`/etc/udev/rules.d` and replugging the device.
+#. In addition, if you don't want to use the root access to run the python script, copy the provided udev rule from the :file:`99-hid.rules` file to the :file:`/etc/udev/rules.d` and reconnect the device.
+#. If you want to connect to a device with a different Vendor or Product ID other than the one specified in the file, you can either run the script if you have the root permission or add a new entry to the :file:`99-hid.rules` file with your Vendor and Product ID. 
```

Comment:
```suggestion
   #. If you want to connect to a device with a different Vendor or Product ID other than the one specified in the file , you can either run the script if you have root permission or add a new entry to the :file:`99-hid.rules` file with your Vendor and Product ID, then copy the provided udev rule from the :file:`99-hid.rules` file to the :file:`/etc/udev/rules.d`  and reconnect the device.
```

Reply:
Just thought that if we give it as a separate sentence it will be applicable for both either and or case which is not true. So, had to rephrase it.